### DISCUSSION TITLE
BBC removing download from computer based iPlayer

### DISCUSSION THREAD

**Discussion by operacentric** [Ascoware/get-iplayer-automator#473] [General]
Hi all - this is one of the handiest apps I know. BBC announced yesterday that they are removing the ability to download programmes from the web-version of iPlayer on 11 March 2024. Is get-iplayer independent of BBC-provided download capability (I'm not tecchie enough to know!)? Or will our world come crashing down on 11 March? The mobile/iPad apps download ability is not affected, apparently, although they are DRM protected and expire after the permitted time.

**Comment by skovatch** [comment]
Right, get_iplayer doesn't rely on anything in the BBC iPlayer application. As I read it, BBC is just no longer supporting their desktop apps for iPlayer, which had the ability to download content for later viewing. 

Given that the mobile app will continue to work as it does now, I don't think there will be any interruption in get_iplayer, and by extension, GiA. It's possible there could be an update to get_iplayer needed, but if there is it shouldn't take long to incorporate it.

**Comment by operacentric** [comment]
Hi Scott

Well, if that is so, that’s great news!

Here was the report I read in the press, which you may well already have seen - https://www.ispreview.co.uk/index.php/2024/02/bbc-iplayer-scrap-offline-tv-show-downloads-for-desktop-users.html

**Reply by skovatch** [reply]
There have been a couple of other comments in bug reports here, and they have all been consistent on the details, so that's encouraging. While I didn't see that particular article, it's consistent with what I read in The Express yesterday.

I wouldn't put it past BBC to make some kind of change to block desktop downloads in general, but it's easy to make it look like your request is coming from a mobile device. Let's see what happens in any event. I'm not too concerned about this development.

**Comment by operacentric** [comment]
Thank ;)
